packages/client-api-schema/src/index.ts

```diff
@@ -27,6 +27,9 @@
  *   "result": {"success": true}
  * }
  * ```
+ *
+ * # Methods
+ * {@link CreateChannel}
  */
```

Comment:
We maintain a list of links on the front page of the API docs
![Screenshot 2020-08-26 at 18 42 32](https://user-images.githubusercontent.com/1833419/91337890-f7cce880-e7cb-11ea-9902-2da9a62e68a9.png)

Just one method shown for now, but we would add all of them (possibly pinching off a browser-only list)

Reply:
These previews look super clean!